src/metadata/countries.json

```diff
@@ -15,7 +15,7 @@
     "dz",
     "Algeria",
     "213",
-    "+213 (..) ... ...."
+    "+213 (...) .. .. .."
```

Comment:
```suggestion
    "+213 (...) ... ..."
```

Reply:
@salahlalami, thank you for your contribution. I agree that area codes in Algeria must have been three digits, but let's not break the general mask structure of the tail, which suggests at least three slots in each component.